packages/components/src/FilterBar/FilterBar.scss

```diff
@@ -1,15 +1,28 @@
 // Filter styles
-$tc-list-filter-input-size: 200px !default;
-$tc-list-filter-highlight-color: #f6e5bb !default;
+$tc-filter-bar-highlight-color: #f6e5bb !default;
+$tc-filter-bar-width: 250px !default;
 
 .filter {
 	display: flex;
+	height: $padding-larger;
+	background-color: rgba($silver, 0.2);
+	color: $dove-gray;
 	align-items: top;
 	padding: 0;
-	background: $white;
+	border-radius: $border-radius-base;
 
 	.search-icon {
+		// margin: $padding-small;
 		margin: $padding-small;
+		margin-right: $padding-smaller;
+		margin-top: 7px; // ...
```

Comment:
why 7px

Reply:
agree, discussed with UI guys, because the height is 35 ... which is not compatible to align with our current padding